File: app/src/main/java/com/yb/uadnd/matchcentre/model/MatchFeedApiInterface.java
package com.yb.uadnd.matchcentre.model;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface MatchFeedApiInterface {

    @GET("{matchId}/commentary")
    Call<Commentary> getMatchCommentary(@Path("matchId") String id);

    @GET("{matchId}")
    Call<Match> getMatch(@Path("matchId") String id);

}

File: app/src/main/java/com/yb/uadnd/matchcentre/model/MatchFeedApiInterface.kt
package com.yb.uadnd.matchcentre.model

import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Path

interface MatchFeedApiInterface {

    @GET("{matchId}/commentary")
    fun getMatchCommentary(@Path("matchId") id: String): Single<Commentary>

    @GET("{matchId}")
    fun getMatch(@Path("matchId") id: String): Single<Match>

}
